packages/sdk/utils/src/typed-data-constructors.ts

```diff
@@ -59,3 +60,43 @@ export const authorizeSigner = ({
     role,
   },
 })
+
+interface AttestationDetails {
+  identifier: string
+  issuer: string
+  account: string
+  signer: string
+  issuedOn: number
+}
+
+export const registerAttestation = (
+  chainId: number,
+  contractAddress: Address,
+  message?: AttestationDetails
+) => {
+  return {
+    types: {
+      EIP712Domain: [
+        { name: 'name', type: 'string' },
+        { name: 'version', type: 'string' },
+        { name: 'chainId', type: 'uint256' },
+        { name: 'verifyingContract', type: 'address' },
+      ],
+      OwnershipAttestation: [
```

Comment:
Is there a reason we don't call this AttestationDetails or vice versa?

Reply:
needs to match the variable name declared in the FA smart contract